vowels-only.js

```diff
@@ -0,0 +1,57 @@
+// // Write a function called vowelsOnly which accepts a string str 
+// // and returns a new string consisting of just the vowels in str.
+
+// function vowelsOnly(str) {
+
+//     var matches = str.match(/[aeiou]/gi);
+//     // console.log("matches", matches);
+
+//     var newStr = "";
+
+//     if (!matches) {
+//         return "";
+//     }
+
+//     for (var i = 0; i < matches.length; i++) {
```

Comment:
I made you do this so you can figure it out, but in reality you would just use the "join" method:
```javascript
return matches.join("");
```

so really your whole function can be:
```javascript
function vowelsOnly(str) {
    const matches = str.match(/[aeiou]/gi);
    return matches ? matches.join("") : "";
}
```

But your solution is great too!